build/teamcity/cockroach/create-docs-issue.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# This script is called by the build configuration
+# "Cockroach > Create Docs Issue" in TeamCity.
+
+set -euxo pipefail
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+source "$dir/teamcity-support.sh"
+source "$dir/teamcity-bazel-support.sh"
+
+BAZEL_SUPPORT_EXTRA_DOCKER_ARGS="-e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e TC_BUILD_BRANCH" run_bazel << 'EOF'
```

Comment:
The list of environment variables in `BAZEL_SUPPORT_EXTRA_DOCKER_ARGS` should be exactly the set of environment variables your command will need to execute correctly. Doesn't look like you need any of these, but you do need `BUILD_VCS_NUMBER` and `GITHUB_API_TOKEN`.

Reply:
Done.